app/views/components/view_application.scala.html

```diff
@@ -19,295 +19,324 @@
+                                    <div>
+                                        <p class="govuk-body govuk-!-font-weight-bold mb-2">@pdf.reference</p>
+                                    </div>
+                                </div>
+                                <div>
+                                    <dl>
+                                        <dt>
+                                            <p class="govuk-body">
                                                                 <span class="govuk-!-font-weight-bold">@messages("application.footer.sendingSamplesTo.heading")
                                                                 </span>
-                                                            </p>
-                                                        </dt>
-                                                        <dd class="govuk-!-margin-left-0">
-                                                            <p class="govuk-body">
-                                                            @Html(messages("confirmation.sendingSamples.address"))
-                                                            </p>
-                                                        </dd>
-                                                    </dl>
-                                                </div>
-                                            }
-                                        </div>
-                                    </div>
+                                            </p>
+                                        </dt>
+                                        <dd class="govuk-!-margin-left-0">
+                                            <p class="govuk-body">
+                                                @Html(messages("confirmation.sendingSamples.address"))
+                                            </p>
+                                        </dd>
+                                    </dl>
+                                </div>
                                 }
                             </div>
-                        }
+                            </div>
                     </div>
+                    }
+                    }
                 </div>
             </div>
         </div>
-        <div class="print-document" id="print-document">
-            <div class="govuk-grid-row print-header">
-                <div class="govuk-grid-column-one-third">
-                    <img class="crown-logo-atar" alt="HM Revenue and Customs" src="@createLink(messages("logo.file.name"))" />
-                </div>
-                <div>
-                    <h2 class="govuk-heading-m govuk-!-margin-top-1 govuk-!-margin-bottom-0 print-header__heading">
-                        @messages("view.application.title.text")
-                    </h2>
-                </div>
-            </div>
+    </div>
+</div>
+<div class="print-document" id="print-document">
+    <div class="govuk-grid-row print-header">
```

Comment:
as discussed there is an issue with the browser print as it is manipulated via css. A simple way is to remove the 
`print-header` from ` <div class="govuk-grid-row print-header">.`

We then add to the `_view_application.scss` file

```
.green-ruling-application {
  text-align: right;
  margin-top: 2em;
  font-size: 1.2em;
  color: #00703C;
  font-weight: 600;
  margin-right: 15px;
}
```

and add it as a class to the h2. 

`            <h2 class="govuk-heading-m govuk-!-margin-top-1 govuk-!-margin-bottom-0 green-ruling-application">
`
on line 200

Reply:
ideally we review and re-write the css but for now this would help patch it up.

<img width="1399" alt="Screenshot 2023-09-07 at 14 46 13" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/f5403a13-c3f6-488d-8a9f-470ccb6460b6">


<img width="1464" alt="Screenshot 2023-09-07 at 14 46 08" src="https://github.com/hmrc/binding-tariff-trader-frontend/assets/47248104/54466a42-be0e-475e-9b31-d76a631ff178">